Sources/ServerConfigPersistenceKey/ServerConfigPersistenceKey.swift

```diff
@@ -0,0 +1,83 @@
+import ApiClient
+import Build
+import ComposableArchitecture
+import Dependencies
+import Foundation
+@_exported import ServerConfig
+
+@propertyWrapper
+public struct ServerConfig_: Equatable {
+  @Shared(.fileStorage(.serverConfig)) var config = ServerConfig()
+
+  public init(config: ServerConfig = ServerConfig()) {
+    self.config = config
+  }
```

Comment:
Just an FYI, but after experimenting with this approach of wrapping some shared state in a property wrapper in my app I did find a serious issue with this initializer - it can trigger an infinite view body render loop if invoked from a view body.

This can be easily done, for example if you have some child feature that uses it:
```swift
@Reducer
struct Child {
  struct State: Equatable {
    @ObservationStateIgnored @ServerConfig_ var config
  }
}
```

Then you initialize the child state in a view (e.g. to pass to `NavigationLink(state:)`), and that view's state also uses the property wrapper, this causes the property wrapper on the child to initialize, which in term updates the shared state, which triggers the parent view to update, which calls `body` and the loop begins.